ims/ostis_tech/unificated_models/section_unificated_models_kb/section_subjdomain_of_logical_formulas_and_logical_ontologies/exercises/exercise_1/def_section_subjdomain_of_logical_formulas_and_logical_ontologies_exercise_1.html

```diff
@@ -0,0 +1 @@
+<p>Формализуйте следующие логические формулы: ((A∧B)～C); ((A∧B)∨C)～((A∧R)∨D).</p>
```

Comment:
Why we need such long file names? It's already in a directory with the same name